sematic/types/types/link.py

```diff
@@ -0,0 +1,17 @@
+# Standard library
+from dataclasses import dataclass
+from urllib.parse import urlparse
+
+
+@dataclass
+class Link:
+    label: str
+    url: str
+
+    def __init__(self, label: str, url: str):
+        parsed_url = urlparse(url)
+        if len(parsed_url.scheme) == 0 or len(parsed_url.netloc) == 0:
+            raise ValueError("Incorrect URL: {}".format(repr(parsed_url)))
```

Comment:
In the spirit of being highly approachable/usable, we might want to specify what's wrong with the URL. E.g. `"Incorrect URL, missing scheme (ex: 'https://'). Provided URL: {}"`. Similarly friendly thing for netloc.